src/api/java/mekanism/api/gear/ModuleData.java

```diff
@@ -86,11 +87,30 @@ public final int getMaxStackSize() {
     }
 
     /**
-     * Exclusive modules only work one-at-a-time; when one is enabled, others will be automatically disabled.
+     * Exclusive modules only work one-at-a-time; when one is enabled, incompatible modules will be automatically disabled.
      *
      * @return {@code true} if this module type is exclusive.
      */
+    @Deprecated(forRemoval = true, since = "10.2.3")
     public final boolean isExclusive() {
+        return isExclusive(ExclusiveFlag.ANY);
+    }
+
+    /**
+     * Exclusive modules only work one-at-a-time; when one is enabled, incompatible modules will be automatically disabled.
+     *
+     * @param mask {@link ExclusiveFlag} mask
```

Comment:
`@param mask Mask of all {@link ExclusiveFlag flags} to check exclusivity against.`